README.md

```diff
@@ -187,55 +187,39 @@ You can send data to HTTP endpoints from actions in your repository, such as ope
+   ![Webhook branch created](assets/webhook-branch-created.png)
 
-Continue to the next section to create a PR workflow. Once you raise a PR, you’ll see the trigger in action on this site.
+Continue to the next section to push a new branch. Once a new branch is pushed, you’ll see the trigger in action on this site.
 
-### PR Workflow
+### Development Workflow
 
 1. For this section, let’s add **developer** user to the project. 
 2. From the left navigation, click on **Members** and then **+ Add Member**. Find **developer** from the **User** dropdown, choose **Contributor** role, and select **Add member to this project**.
 
-![Add developer to devdays project](assets/add-developer-member.png)
+   ![Add developer to devdays project](assets/add-developer-member.png)
 
 3. On the Gitness portal, log out from this profile by clicking the icon on the bottom-left corner and log back in using the developer credentials. 
 4. Back in VS Code, create a new branch named **update-app-version** and commit a change. For instance, this commit updates the `pkg/version/version.go` file, resulting in the application version changing from **6.6.1** to **6.6.2**.
 
-![Update image tag](assets/update-image-tag.png)
+   ![Update image tag](assets/update-image-tag.png)
 
 5. When you commit and push changes, you’ll be prompted to enter username and password for the remote repository. On the Gitness console, click **Generate Clone Credential** under **Git Clone URL** and use the generated credentials.
 
-![Generate Clone Credentials](assets/generate-clone-creds.png)
+   ![Generate Clone Credentials](assets/generate-clone-creds.png)
 
-Once the push is complete, you should see the remote repository reflect the changes.
+   Once the push is complete, you should see the remote repository reflect the changes, and a response on webhook.site for this event.
 
-6. Click on **Pull Requests** from the left navigation menu and then click **+ New Pull Request**. Select your feature branch as the source branch, add a PR description, and click **Create Pull Request**. 
+6. Navigate to the webhook.site dashboard and you should see the POST request details triggered by your PR. 
 
-![Create PR](assets/create-pr.png)
-
-7. Click **Add +** under Reviewers and find **Administrator** from the list.
-
-8. Navigate to the webhook.site dashboard and you should see the POST request details triggered by your PR. 
-
-![Webhook triggered PR created](assets/webhook-site-pr-created.png)
-
-> [!NOTE]
-> Switch from the **developer** account to the **admin** account before moving to the next step, as the developer account doesn’t have permission to edit the webhook.
-
-9. Toggle the webhook off from the podinfo repository under **Webhooks** → **trigger_on_pr**.
-
-![Disable Webhook](assets/disable-webhook.png)
-
-> [!CAUTION]
-> **Don't merge the PR yet**. You’ll do it later at the CI/CD section. 
+   ![Webhook triggered branch creation](assets/webhook-site-branch-created.png)
 
 ### Secret Scanning
```

Comment:
Before this section, the user needs to swap the developer account with administrator account so that they can enable secret scanning, create pipelines, etc in subsequent sections.

Reply:
I have added steps to log in as admin before this, then steps to log back in as the developer after